docs/source/registrationdetails.rst

```diff
@@ -0,0 +1,56 @@
+Registration Details
+=====================
+
+The determination of uniqueness is the core of any chemical registration system, 
+meaning the method to decide weather two molecular structures are the same or not.
+lwreg uses a hash-based approach to identify duplicates. 
+As all comparisons are simply string comparisons, lwreg is a fast approach in any modern database system.
+
+Methodology of Molecule Hash Computation
+----------------------------------------
+lwreg makes uses of the RDKit's RegistrationHash functionality. 
```

Comment:
Maybe link to the documentation for this?
https://rdkit.org/docs/source/rdkit.Chem.RegistrationHash.html